tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -285,16 +318,51 @@ public static IList<TestEndpoints> GetAllTestEndpoints()
                     new ("basketservice", "Application started"),
                     new ("postgres", "database system is ready to accept connections"),
                 ]),
+            // Test endpoints disabled for AppHostRunsCleanlyForAppsWithNoOtherTests as they have specific tests
+            new TestEndpoints("WithDockerfile.AppHost", []),
+            new TestEndpoints("KafkaBasic.AppHost", []),
+            new TestEndpoints("AzureFunctionsEndToEnd.AppHost", [])
         ];
 
         return candidates;
     }
 
+    private static async Task TestEventHubsAppHost(DistributedApplication app, string appHostPath, ITestOutputHelper testOutput)
+    {
+        using var client = CreateHttpClientWithResilience(app, "api");
+
+        var consumerMessage = "Hello, from /test sent via producerClient";
+        var consumerMessageLoggedTask =
+                app.WaitForTextAsync(log => log.Contains(consumerMessage), resourceName: "consumer")
+                    .WaitAsync(TimeSpan.FromMinutes(2))
+                    .ConfigureAwait(false);
+
+        var path = "/test";
+        testOutput.WriteLine($"*** TestEventHubsAppHost calling {path} endpoint");
+
+        var response = await client.GetAsync(path);
+        Assert.True(HttpStatusCode.OK == response.StatusCode, $"Endpoint '{client.BaseAddress}{path.TrimStart('/')}' for resource 'consumer' in app '{Path.GetFileNameWithoutExtension(appHostPath)}' returned status code {response.StatusCode}");
+
+        try
+        {
+            await consumerMessageLoggedTask;
+        }
+        catch (TimeoutException te)
+        {
+            throw new XunitException($"Timed out waiting for the consumer message to be logged: '{consumerMessage}'", te);
+        }
+    }
+
     public static TheoryData<TestEndpoints> TestEndpoints()
     {
         TheoryData<TestEndpoints> theoryData = new();
         foreach (var candidateTestEndpoint in GetAllTestEndpoints())
         {
+            if (candidateTestEndpoint.Skip)
```

Comment:
Why do we have test endpoints declared that should be skipped?

Reply:
These have specific tests in `ProjectSpecificTests.cs`, so don't need this simple "does the app at least run cleanly".